app-backend/api/src/main/scala/campaign/Routes.scala

```diff
@@ -54,22 +54,30 @@ trait CampaignRoutes
                 cloneCampaign(campaignId)
               }
             }
-          } ~ pathPrefix("permissions") {
-          pathEndOrSingleSlash {
-            get {
-              listCampaignPermissions(campaignId)
-            } ~
-              put {
-                replaceCampaignPermissions(campaignId)
-              } ~
-              post {
-                addCampaignPermission(campaignId)
-              } ~
-              delete {
-                deleteCampaignPermissions(campaignId)
+          } ~
+          pathPrefix("clone-owners") {
```

Comment:
this sounds like a verb -- I was surprised that it was a `get` endpoint, since it sounded like it was for cloning the owners of a campaign

Reply:
I agree with this, but I struggled to think of a good name so this was the result. Any ideas?